### DISCUSSION TITLE
Does you need a chrome cast device?

### DISCUSSION THREAD

**Discussion by anthnyajp** [JohnStrunk/wahoo-results#297] [Q&A] [answered]
I haven't tried this yet, but do you need to have a chrome cast device or can I just use an HDMI cable to external monitor / TV to display the scoreboard?

**Comment by JohnStrunk** [comment, marked as answer]
The early versions of WR assumed an HDMI connection to a TV, but at this point, it's Chromecast-only.
You can test it out w/o actually having a CC--- The scoreboard contents will appear in the window on the [run screen](https://wahoo-results.readthedocs.io/en/stable/configuration.html#config-run-tab).

Note that the current generation "Chromecast w/ GoogleTV" require [some configuration](https://wahoo-results.readthedocs.io/en/stable/chromecast-compat.html) to work well, but any chromecast will work fine.
